components/event_dispatcher/introduction.rst

```diff
@@ -496,7 +496,7 @@ Eventi e ascolatori consapevoli del distributore
 ``EventDispatcher`` inietta sempre l'evento distribuito, il nome dell'evento
 e un riferimento a sé stesso agli ascoltatori. Questo può portare ad applicazioni
 avanzate per ``EventDispatcher``, incluse la possibilità per gli ascoltatori di distribuire altri eventi,
-il concatenamento degli eventi o anche il caricamento pigro di più ascoltatori
+il concatenamento degli eventi o anche il caricamento pigro ("lazy") di più ascoltatori
 nell'oggetto distributore. Ecco degli esempi:
```

Comment:
Mi sembra che 'caricamento pigro' suoni alquanto strano in italiano.
Metterei, la prima volta che se ne parla, tra parentesi: "lazy loading".

Il giorno 21 marzo 2015 13:26, Massimiliano Arione <notifications@github.com
> ha scritto:

> In components/event_dispatcher/introduction.rst
> <https://github.com/garak/symfony-docs-it/pull/206#discussion_r26891469>:
>
> > @@ -496,7 +496,7 @@ Eventi e ascolatori consapevoli del distributore
> >  ``EventDispatcher`` inietta sempre l'evento distribuito, il nome dell'evento
> >  e un riferimento a sé stesso agli ascoltatori. Questo può portare ad applicazioni
> >  avanzate per ``EventDispatcher``, incluse la possibilità per gli ascoltatori di distribuire altri eventi,
> > -il concatenamento degli eventi o anche il caricamento pigro di più ascoltatori
> > +il concatenamento degli eventi o anche il caricamento pigro ("lazy") di più ascoltatori
> >  nell'oggetto distributore. Ecco degli esempi:
>
> perché aggiungere "lazy" qui?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/garak/symfony-docs-it/pull/206/files#r26891469>.
>